### ISSUE TITLE
pig pen

### ISSUE THREAD

**Issue by ne1411** [taberenc/AOG_2017#4]
Seems to trap you against the fence if a pig is near you and you try to move around the pig against the fence.

**Comment by taberenc** [comment]
screen capture, please. there are 2 pig farms if i remember correctly. and 2 fences for each pig farm.

**Comment by f4s1k1n** [comment]
I think it was in first town... Feed the pig quest...

**Comment by ne1411** [comment]
Yes that’s the one, think it happens when the pig moves near the fence and if you move the char at the same moment the pig does and it moves towards the fence as you are passing to get out , it seems to push you into the fence and you get trapped .



Sent from myMail for iOS


Saturday, 14 October 2017, 1:41 pm +0100 from notifications@github.com <notifications@github.com>:

I think it was in first town... Feed the pig quest...
